multi-server-adding-mailbox.adoc

```diff
@@ -0,0 +1,228 @@
+[[multi-server-adding-mailbox]]
+= Adding a Mailbox Server to a Single Server Configuration
+
+In the {product-name} ({product-abbrev}) single server environment,
+the LDAP, MTA, and mailbox services are on one machine. This chapter
+explains how to add a new machine that is configured as a mailbox
+server to a single server configuration and how to remove the mailbox
+server from the single server node.
+
+== Setup Requirements For Adding a Mailbox Server
+
+* The new machine you are adding must have the same operating system,
+  including the latest version and patch levels, as installed on the
+  single server.
+* The system clock must be configured with the same time on both
+  machines.
+* You must install the same version of the {product-abbrev} software
+  that is installed on the single server node.
+* A copy of the {product-abbrev} license needs to be added to a
+  directory on the new machine.
+* You are adding {product-short} Proxy to {product-abbrev}, this
+   should be installed on the existing single-server before you set up
+   the new mailbox server. See <<_installing_zimbra_proxy>>.
+
+
+== Overview of Process
+
+* {product-short} Mailbox Server is installed on the prepared machine.
+* Customized configuration for the single-server, such as custom themes
+  and Zimlets are added to the new mailbox server.
+* Commercial SSL certificates are added to the new mailbox server.
+* User accounts are moved from the single server to the new mailbox
+  server.
+* If you are moving all accounts from the single server, the mailbox
+  server is stopped on the single server machine.
+
+
+== Configuring the Mailbox Server
+
+The host name and zmhostname configured on the mailbox server is the
+same as on the name on the single server.
+
+Make sure you know the LDAP master password as you configure it on the
```

Comment:
If you are installing zmproxy or MTA on the second node, you will also need to write down "Bind password for postfix ldap user", "Bind password for amavis ldap user" and "Bind password for nginx ldap user" - all these are already in LDAP if these services have been installed on the first node.